lib/Runtime/Language/SourceTextModuleRecord.h

```diff
@@ -47,15 +47,18 @@ namespace Js
 
         void SetSpecifier(Var specifier) { this->normalizedSpecifier = specifier; }
         Var GetSpecifier() const { return normalizedSpecifier; }
+        const char16 *GetSpecifierSz() const { return JavascriptString::FromVar(this->normalizedSpecifier)->GetSz(); }
```

Comment:
Currently it is used in debug-only traces, but it can be used in test-build traces too.

---
In reply to: [110757106](https://github.com/Microsoft/ChakraCore/pull/2822#discussion_r110757106) [](ancestors = 110757106)